apps/docs/turbo.json

```diff
@@ -4,8 +4,7 @@
   "pipeline": {
     "build": {
       "dependsOn": ["^build", "prebuild"],
-      "outputs": ["dist/**"],
-      "inputs": ["src/**", ".vitepress/**", "types/**"]
```

Comment:
We must invalidate the cache when things inside the `.vitepress` or `types` directory change. Otherwise, re-building the docs after changing something inside these directories won't do anything. (cache-hit)

> cc @Torres-ssf

Reply:
Good point. I'm going to close this and revisit. The more complete fix is to probably move the auto generated content outside of `src` and into `.typedoc`.